api/v1alpha1/podhttpchaos_types.go

```diff
@@ -26,14 +26,20 @@ type PodHttpChaosSpec struct {
 
 // PodHttpChaosStatus defines the actual state of PodHttpChaos.
 type PodHttpChaosStatus struct {
+	//+kubebuilder:deprecatedversion:warning="Pid is deprecated, please use Uid instead"
 	// Pid represents a running tproxy process id.
 	// +optional
 	Pid int64 `json:"pid,omitempty"`
```

Comment:
```suggestion
	//+kubebuilder:deprecatedversion:warning="Pid is deprecated, please use Uid instead"
	// Pid represents a running tproxy process id.
	// Deprecated: Pid is deprecated, please use Uid instead
	// +optional
	Pid int64 `json:"pid,omitempty"`
```

we should also add a godoc style "Deprecated: " comment.

Reply:
and `+kubebuilder:deprecatedversion:warning` marker is supported by `controller-gen > 0.6.0`;  we should upgrade it in dev-env;

see https://github.com/kubernetes-sigs/controller-tools/pull/541 and https://github.com/kubernetes-sigs/controller-tools/releases/tag/v0.6.0

Reply:
The godoc-style comments are appended. It seems there are breaking changes in controller-gen v0.6.0, I'd like to upgrade it in another PR.